README.md

```diff
@@ -13,12 +13,12 @@ Bugs and feature requests can be reported through the [Trac issue tracker](https
 
 You can submit your changes through a [pull request](https://github.com/boostorg/graph/pulls). One of the maintainers will take a look (remember that it can take some time).
 
-There is no mailing-list specific to Boost Graph, although you can use the general-purpose Boost [mailing-list](https://groups.google.com/forum/#!forum/boost-devel-archive) using the tag [graph].
+There is no mailing-list specific to Boost Graph, although you can use the general-purpose Boost [mailing-list](http://lists.boost.org/mailman/listinfo.cgi/boost-users) using the tag [graph].
```

Comment:
Weird, we wrote this README not so long ago 😕 
And you are right: the google group boost-devel-archive seems to have disappeared! (although being still in google results)